staging/src/k8s.io/apiserver/pkg/storage/etcd3/lease_manager.go

```diff
@@ -103,13 +109,13 @@ func (l *leaseManager) GetLease(ctx context.Context, ttl int64) (clientv3.LeaseI
 	if err != nil {
 		return clientv3.LeaseID(0), err
 	}
+	if l.leaseAttachedObjectCount > largeLeaseThreshold {
```

Comment:
Is there a downside to just using 5k as the value and deleting this check completely?

Reply:
This log is used to investigate some test problems and I think it has not been completed yet.

Reply:
It was needed to get the max count in our tests, which is 50-70k objects per lease. We know this already so we can remove this log.